app/controllers/lettings_logs_controller.rb

```diff
@@ -51,7 +53,11 @@ def show
   def edit
     @log = current_user.lettings_logs.find_by(id: params[:id])
     if @log
-      render "logs/edit", locals: { current_user: }
+      if @log.unresolved
+        redirect_to(send("lettings_log_#{@log.form.get_question('startdate', @log).page.id}_path", @log))
```

Comment:
```suggestion
        redirect_to(send("lettings_log_#{@log.form.get_question('startdate', @log).page.id}_path", @log))
```

Do we have a helper to do this? Please could you create one, if not?

Reply:
Not a helper, but this is now a bit smaller